### PR TITLE
Update Exotel secrets for qa

### PR DESCRIPTION
**Pull request by qptr**
We have been getting [these Sentry alerts](https://resolve-to-save-lives.sentry.io/issues/3796240203/?project=1217715&referrer=slack) every Thursday since December 2022.

It has been happening whenever the exotel_tasks rake task runs the UpdatePhoneNumberDetailsWorker to enqueue to Sidekiq. The ExotelApiService eventually needs valid, non-empty SID and token. These values are picked up from the environment, and the qa secrets for these variables were blank. This PR copies over these values from Sandbox.

### CODE REVIEW

**Review by roypeter — APPROVED** [review]

**Review by prabhanshuguptagit — APPROVED** [review]
Tested the readme locally. Works ✨

File: doc/SecretManagement.md [hunk 1]
```diff
@@ -7,32 +7,47 @@ Kubeseal: Encrypts and decrypts secrets at k8s level. Kubeseal does not allow lo
...
 
 # Commit the changes
 ```
 
-`decrypt` , `encrypt` and `kubeseal_encrypt` supports multiple options to decrypt/encrypt secrets in a all the environments or a specific environment or a specific file. Please refer the usage section for more details.
+`decrypt` , `encrypt` and `kubeseal_encrypt` supports multiple options to decrypt/encrypt secrets in all environments or a specific environment or a specific file. Please refer to the usage section for more details.
+
+## Secrets deployment
+
+The secrets file is auto-synced on [ArgoCD](https://argocd-qa.simple.org/applications/argocd/secrets?view=tree&resource=&node=bitnami.com%2FSealedSecret%2Fsimple-v1%2Fsimple-server%2F0) (find the login credentials on the Simple password manager). To deploy manually, directly update the secrets (don't forget to base64 encode them!) in the simple-server live manifest on ArgoCD and hit `sync`.
```
Comment (roypeter) [inline doc/SecretManagement.md:53]: To deploy manually, directly update the secrets (don't forget to base64 encode them!) in the simple-server live manifest on ArgoCD and hit `sync`.  is not recommended all changes should be in git

Comment (qptr) [reply doc/SecretManagement.md:53]: @prabhanshuguptagit ^

Comment (prabhanshuguptagit) [reply doc/SecretManagement.md:53]: :shipit:

**Pull request merged**